language/tools/move-cli/src/sandbox/commands/test.rs

```diff
@@ -334,7 +334,7 @@ pub fn run_one(
     }
 
     let expected_output = fs::read_to_string(exp_path).unwrap_or_else(|_| "".to_string());
-    if expected_output != output {
+    if expected_output.trim() != output.trim() {
```

Comment:
Why was this change needed?

Reply:
@tnowacki When I was creating the test, I copied the result from the console along with not-display symbol that the editor does not display and could not delete it.
`format_diff(...)` also didn't display the difference.
Therefore, to simplify writing tests, I added `.trim()`

Reply:
@tnowacki Remove it?

Reply:
Sorry missed the mention!

I'd prefer to have it removed. We have `UB=1` or `UPBL=1` to update the baseline for these tests to prevent any copy paste issues (and to make your life easier :) )

Reply:
@tnowacki Fixed